src/styles/_catalog.scss

```diff
@@ -13,7 +15,6 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
   &__card:not(:nth-child(4n)) {
```

Comment:
We don't need pseudo-class   &__card:not(:nth-child(4n))

Reply:
i fixed it